### ISSUE TITLE
Inklecate crashes when parenthesis are left off of an external function declaration

### ISSUE THREAD

**Issue by nullObjectPtr** [inkle/ink#58]
Reproducable on version 0.2.3 alpha, using windows version of inklecate. Haven't tried with the mac version.

This is the story I used to reproduce:
```
EXTERNAL bring_the_pain

== CrashMe ==
This is gonna hurt...
+ DONE
```

Stack trace as follows.
```
Unhandled Exception: System.NullReferenceException: Object reference not set to an instance of an object.
   at Ink.InkParser.ExternalDeclaration()
   at Ink.StringParser.ParseObject(ParseRule rule)
   at Ink.StringParser.OneOf(ParseRule[] array)
   at Ink.InkParser.StatementAtLevel(StatementLevel level)
   at Ink.InkParser.<StatementsAtLevel>c__AnonStorey6.<>m__0()
   at Ink.StringParser.ParseObject(ParseRule rule)
   at Ink.StringParser.Interleave[T](ParseRule ruleA, ParseRule ruleB, ParseRule untilTerminator, Boolean flatten)
   at Ink.InkParser.StatementsAtLevel(StatementLevel level)
   at Ink.InkParser.Parse()
   at Ink.CommandLineTool.<CommandLineTool>c__AnonStorey1.<>m__1()
   at Ink.CommandLineTool.TimeOperation(String opDescription, Action op)
   at Ink.CommandLineTool..ctor(String[] args)
   at Ink.CommandLineTool.Main(String[] args)
```

**Comment by joethephish** [comment]
Oops, thanks!

**Comment by joethephish** [comment]
Fixed in d53e66365fddccbaf13e1b07976356f9d70fe9cc on development branch.

**Comment by nullObjectPtr** [comment]
Thanks Joseph!
Much appreciated.

-Jonathan
